src/server.js

```diff
@@ -146,16 +156,11 @@ function index(req, res, next) {
 		pkg,
 		Helper.config
 	);
+	if (!data.theme.includes(".css")) { // Backwards compatibility for old way of specifying themes in settings
```

Comment:
If memory serves, I think `#includes` is not available before Node.js v... 6? I think that on v5 it was behind a flag, and did not exist on v4 or something like that.

Reply:
That's contradicted by http://node.green/#ES2015-built-in-extensions-String-prototype-methods-String-prototype-includes

Reply:
That's right, I guess I was looking at the wrong section (it says `.includes` with symbols is not in before Node v6), my bad.
Also, last time we probably hit that when we were still supporting Nove v0.12.
So all good here, thanks!